src/main/java/org/fundacionjala/coding/mary/EANValidator.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.mary;
+
+public final class EANValidator {
+    private static final int MOD_10 = 10;
+    private static final int TOTAL = 10;
+    private static final int MULTIPLIED_BY_3 = 3;
+
+    private EANValidator() {
+    }
+
+    public static boolean validate(final String eanCode) {
+        double validateCode = (Double.parseDouble(eanCode)) % MOD_10;
+        return (validateCode == checksum(eanCode)) ? true : false;
```

Comment:
the checksum should be compare with the last digit in the eanCode.
```java
return Character.getNumericValue(eanCode.charAt(eanCode.length() - 1)) == checksum(eanCode);
```

Reply:
Hi Carlos, 
Actually I'm getting the last digit of the code in the "validateCode" variable by applying the module 10 of the number in line 12. I guess the name of my variable was not good.
Thanks for the suggested code, I will use that to update my code.